lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -603,13 +769,6 @@
     restart_policy_delay: 5
     restart_policy_window: 30
 
-- name: Set placement preferences
```

Comment:
Maybe instead of removing this example, change it to use the new syntax? Or doesn't that make sense?

Reply:
Also fixed the example for `restart_config`.